### ISSUE TITLE
NVDA and the mail app for windows10

### ISSUE THREAD

**Issue by bedenhofer** [nvaccess/nvda#8795]
When you open a message in the mail app to read it, you can read the message! but when you hit alt f4, to close the mail app, NVDA crashes, and you half to relaunch it with alt control N! my friend Bobby Vinton, who's an NVDA user as well, reported this to me, a while back, so therefore, I reported this bug a couple of times, and I've gotten responses, but no one has resolved it, whatsoever.

**Comment by DrSooom** [comment]
@bedenhofer: Please give us more details by completely filling out the draft. Thanks.

**Comment by bedenhofer** [comment]
I'm running NVDA version 2018.31, with windows10, and this problem occurs when I restart my PC,

**Comment by Adriani90** [comment]
This is not reproducible on NVDA 2019.3 Beta 2. @bedenhofer can you confirm?

**Comment by Adriani90** [comment]
No updates from the initial author since one and a half years, I am closing this as works for me and abandoned. I cannot reproduce with NVDA 2020.1.
